src/components/carousel/index.js

```diff
@@ -0,0 +1,206 @@
+import React from "react"
+
+import classnames from "classnames"
+import useCurrentWitdh from "../../utils/useCurrentWitdh"
+
+//
+// images: array of image. { url, title }
+//
+export default function Carousel({ images = [], height, onClickRoot }) {
+  const [width, setWidth] = React.useState(0)
+  const [isHoverMode, setIsHoverMode] = React.useState(false)
+  const [currentIndex, setCurrentIndex] = React.useState(0)
+
+  const windowWidth = useCurrentWitdh()
+
+  const nbImages = images.length
+
+  const rootRef = React.useRef(null)
+
+  //
+  // set isHoverMode to true and hide it after some time
+  //
```

Comment:
nit,  we're using one line comments

```
// comment
```

Reply:
I have the habit of using 

```
//
//
//
```

to seperate sections on my code. Should I remove all the triple to a single one? In this case how can I make clear that we enter a new section and not just a commented code?